src/components/summary-list/index.md

```diff
@@ -99,6 +99,8 @@ You can add card actions in the header, which will be shown after the summary ca
 
 For example, if you have multiple rows with "change" actions that all take the user to the same place, you can show a single “change” card action instead. This helps avoid repeating the same row action on every row.
 
+Card actions are bolded to make them visually distinct from row actions. Since card actions can affect the whole summary card, it’s useful to be able to tell them apart from row actions, especially when navigating with a screen magnification tool.
```

Comment:
Maybe 'Card actions use bold text to make them...'? Not sure 'bolded' is a real word, or at least clear to all.